agent/README.md

```diff
@@ -200,14 +206,32 @@ The agent's REST API has the following endpoints:
 }
 ```
   
-- `[POST] /test/start/{testPath}` Signals to the agent that the test with the given testPath is about to start.
-- `[POST] /test/end/{testPath}` Signals to the agent that the test with the given testPath has just finished.
+- `[POST] /testrun/start` If you configured a connection to Teamscale via the `teamscale-` options, this will fetch
+  impacted tests from Teamscale and return them in the response body. The format is the same as returned by Teamscale
+  itself. You must provide a list of all available test cases in the body of the request. These will also be used to
+  generate the test-wise coverage report in `[POST] /testrun/end`. The format of the request body is:
+  
+```json
+[
+    {
+      "clusterId": "<ID of the cluster the test belongs to>",
+      "uniformPath": "<Unique name of the test case>",
+      "sourcePath": "<Optional: Path to the source of the test>",
+      "content": "<Optional: Value to detect changes to the test, e.g. hash code, revision, ...>"
+    }
+]
+```
```

Comment:
We should probably also document the `include-non-impacted` and `baseline` options here